app/src/components/staking/PageValidators.vue

```diff
@@ -11,36 +11,48 @@
       />
     </div>
     <template v-else slot="managed-body">
-      <div class="filterOptions">
+      <div class="filterContainer">
         <TmField
           v-model="searchTerm"
           class="searchField"
           placeholder="Search"
         />
-        <div class="toggles">
-          <TmBtn
-            value="Popular"
-            class="btn-radio secondary"
-            :type="popularSort ? `active` : `secondary`"
-            @click.native="defaultSelectorsController(`popularSort`)"
-          />
-          <TmBtn
-            value="All"
-            class="btn-radio secondary"
-            :type="allValidators ? `active` : `secondary`"
-            @click.native="defaultSelectorsController(`allValidators`)"
-          />
-          <TmBtn
-            value="Active"
-            class="btn-radio secondary"
-            :type="activeOnly ? `active` : `secondary`"
-            @click.native="defaultSelectorsController(`activeOnly`)"
-          />
+        <div class="filterOptions">
+          <div class="toggles">
+            <TmBtn
+              value="Popular"
+              class="btn-radio secondary"
+              :type="popularSort ? `active` : `secondary`"
+              @click.native="defaultSelectorsController(`popularSort`)"
+            />
+            <TmBtn
+              value="All"
+              class="btn-radio secondary"
+              :type="allValidators ? `active` : `secondary`"
+              @click.native="defaultSelectorsController(`allValidators`)"
+            />
+            <TmBtn
+              value="Active"
+              class="btn-radio secondary"
+              :type="activeOnly ? `active` : `secondary`"
+              @click.native="defaultSelectorsController(`activeOnly`)"
+            />
+          </div>
+          <div class="show-mobile-sorting">
```

Comment:
Is the idea that this is always visible or only on mobile?

Reply:
Nope, now it shows only in > 768px wide screens.